Dockerfile

```diff
@@ -22,10 +22,10 @@ RUN ~/src/docker/install.sh
 
 FROM deps
 
-RUN mkdir ~/src
-COPY --chown=carma . /home/carma/src/
-RUN ~/src/docker/checkout.sh
-RUN ~/src/docker/install.sh
+#RUN mkdir ~/src
+#COPY --chown=carma . /home/carma/src/
+#RUN ~/src/docker/checkout.sh
+#RUN ~/src/docker/install.sh
```

Comment:
Those were the changes JB suggested to allow the repo to build properly. That same block of code is already executed on lines 18-21, and the duplicate code block was giving me build errors since it was trying to make the source directory again as seen in the screenshot.
![Screenshot (32)](https://user-images.githubusercontent.com/52839028/88341214-2f4aff80-cd0b-11ea-98f4-9d47a3a1c49c.png)

![image (2)](https://user-images.githubusercontent.com/52839028/88341170-19d5d580-cd0b-11ea-8d29-3490061af4c5.png)